src/chronometer.js

```diff
@@ -1,33 +1,43 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime +=1
+      //printTimeCallback();
```

Comment:
Aqui tem um porém!
Se ele não receber nada como argumento, o `printTimeCallback` é `undefined`.
```js
undefined(); // gera um ERRO (TypeError)!
```
então o ideal é q se faça uma verificação se é uma função!
```js
if (typeof printTimeCallback === 'function') {
  printTimeCallback();
}
```
Assim ele executa a função de callback apenas se ela for uma função!